types/theo/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for theo 8.0
+// Project: https://github.com/salesforce-ux/theo
+// Definitions by: Pete Petrash <https://github.com/petekp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { Iterable, Collection, Map, List, OrderedMap } from "immutable";
+
+export type StyleProperty =
+    | "name"
+    | "value"
+    | "type"
+    | "originalValue"
+    | "category"
+    | "comment"
+    | "meta"
+    | string;
```

Comment:
What is the point of specifying values if all of them are overridden by 'string'?